snd-server/migrations/000_create_users.sql

```diff
@@ -1,5 +1,6 @@
 create table if not exists users (
   id integer primary key,
+  username text not null,
   email text not null,
-  access_key_hash text
+  hashed_password text
 );
```

Comment:
Do we want to ask for both email & username? I'd make one of them optional at least or remove altogether for now?